pkg/controller/acmeorders/sync.go

```diff
@@ -706,26 +706,25 @@ func getAltCertChain(ctx context.Context, cl acmecl.Interface, certURL string, p
 	if err != nil {
 		return false, nil, fmt.Errorf("error listing alternate certificate URLs: %w", err)
 	}
-	// Loop over all alternative chains
```

Comment:
The function is still called `getAltCertChain`, which does not fit the new behavior anymore, I think.

Reply:
changed term to `preferred` @ https://github.com/cert-manager/cert-manager/commit/94509d0490be10953fd97086260d6c37acd8c0c0